app/components/ViewContainer.js

```diff
@@ -44,10 +44,10 @@ export default class ViewContainer extends Component {
     );
   }
 
-        // <AR locs={ this.state.currLoc }/>
   render() {
     return (
       <View>
+        <AR currLoc={ this.state.currLoc } pins={ this.props.pins.pins } />
         <Map 
           dropPin={this.props.actions.getLocationToSave}
```

Comment:
note: this will be changed to the destructured version const {actions} = this.props ..etc
this change is in my following PR